hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/BasicResponse.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.recon.api.types;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * HTTP Response wrapped for a 'basic' request.
+ * 'type': the namespace the request path is on.
+ * 'bucket': Total number of buckets for volume, 0 for other types.
+ * 'dir': Total number of directories for a bucket or directory, 0 for others.
+ * 'key': Total number of keys for a bucket or directory, 0 for others.
+ * 'pathNotFound': set to true if request path is valid
+ */
+public class BasicResponse {
+  @JsonProperty("type")
+  private EntityType entityType;
+
+  @JsonProperty("bucket")
+  private int totalBucket;
+
+  @JsonProperty("dir")
+  private int totalDir;
+
+  @JsonProperty("key")
+  private int totalKey;
+
+  @JsonProperty("pathNotFound")
+  private boolean pathNotFound;
```

Comment:
An arguably better approach is to rename this to `status`, and change the type to an enum, such as `BasicResponseStatusType`.

`PATH_NOT_FOUND` can then be one of the statuses.

Just for reference:
```
public enum BasicResponseStatusType {
  OK,
  PATH_NOT_FOUND
}
```

When reusing `BasicResponseStatusType` for other responses such as the ones below we can use more general name for this enum and add more fields to it.